graphql/community/communityAccountQuery.graphql

```diff
@@ -6,10 +6,14 @@ query communityAccount($address: ID!) {
       uri
     }
     categoryScoreChanges {
```

Comment:
may also want to pick off `ipfsLink` for when IPFS stuff is done, but can add later